tests/integration/components/pikaday-input-test.js

```diff
@@ -599,6 +612,23 @@ module('Integration | Component | pikaday-input', function(hooks) {
     );
   });
 
+  test('it clears input value if initial value is an invalid date ', async function(assert) {
+    assert.expect(3);
+
+    await render(hbs`{{pikaday-input id="pikadayTest" clearInvalidDate=true}}`);
+
+    await click('input');
+    await fillIn('input', 'difouhbadsilufb33');
+
+    assert.equal(this.$('input').val(), 'difouhbadsilufb33');
+
+    await closePikaday();
+
+    assert.equal(this.get('value'), null);
```

Comment:
I'm not sure that this would have been set anywhere -- what are you trying to assert against?

If you want to ensure that the user of Pikaday received `null` after the picker was closed, I would look at other tests that are validating how `onSelection` is invoked using Testdouble. It would be something like this:

```javascript
const onSelection = td.function();
this.set('onSelection', onSelection);

await render(hbs`
  {{pikaday-input onSelection=(action onSelection)}}
`);

assert.verify(onSelection(null));
```